requirements.txt

```diff
@@ -23,6 +23,7 @@ en-core-sci-lg @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.
 h5py==3.2.1
 ipython==7.22.0
 ipywidgets==7.6.3
+lxml==4.6.3
```

Comment:
I guess we want the `requirements.txt` to be sorted alphabetically, right?

Reply:
Good catch! Done in 580adf9.